rainbow_logging_handler/__init__.py

```diff
@@ -63,6 +63,7 @@ def __init__(
         self, stream,
 
         datefmt='%H:%M:%S',
+        passthrough=False,
```

Comment:
The name `passthrough` seems not understandable enough.
How about `ignore_tty_check` ?